homeassistant/components/history/__init__.py

```diff
@@ -426,29 +438,26 @@ def __init__(self) -> None:
         self.included_domains: list[str] = []
```

Comment:
This class really should be moved to `recorder/models.py` since its used in both logbook and history and the typing is currently `Any` in a lot of places because it would be a circular import

Reply:
We can work around circular imports for typing using `from typing import TYPE_CHECKING`  workaround.

Reply:
Definitely can work around the circular imports, but I'd rather not have logbook import from history so I think its better to move it.